### DISCUSSION TITLE
Easy way to remove certain OSM objects?

### DISCUSSION THREAD

**Discussion by jatech1** [onthegomap/planetiler#886] [Q&A]
Is there any easy way, via the custom yaml or other method, to simply build the full default OSM planet, but exclude a couple objects... like house numbers?  I was looking for some default yaml that builds the default, as if no yaml was specified on the command line... then to remove from that, but does that exist?  Or is there a better way?  Thanks!

**Comment by JesseCrocker** [comment]
There is a command line option, `--exclude_layers` that can be used to exclude layers.

**Reply by jatech1** [reply]
Thank you... I do recall seeing that now that you mentioned it.
I assume if I want to be any more selective in removing things, like keeping admin_level=4 state lines in US/Can, but removing from the rest of the world, that I'd need to go to java level?  I am guessing even the yaml could not do that?

**Reply by msbarry** [reply]
Thanks @JesseCrocker! Yes at the moment you can do coarse-grain filtering by layer but you can't get more detailed than that at the moment without forking/modifying the Java or yaml profile. Feel free to submit a feature request or PR. It seems like ideally you could compose multiple profiles together and filter features or attributes from the output. Although I'm not sure what the best syntax to express that would look like...?
